scripts/devaccess/devaccess_wrap.py

```diff
@@ -289,6 +289,19 @@ def oc_get_pods(occmd):
 
         return results
 
+    @staticmethod
+    def oc_get_routes(occmd):
+        ''' provide 'oc get routes' '''
+
+        n_cmd = occmd.normalized_cmd()
+        run_cmd = WhitelistedCommands.oc_cmd_builder(n_cmd)
+        # -o yaml and -o json output of routes can expose
```

Comment:
Can you provide a specific command that will expose a cert.

The public part of the cert is, obviously, public and may be shared.  I'd like to see the specific output to clarify how diligently it should be redacted.